byelaws/article_ii.md

```diff
@@ -0,0 +1,147 @@
+## ARTICLE II: Leadership Team
+
+## Section 1: Composition
+
+There shall be a Leadership Team consisting of not less than six (6) members.
+
+## Section 2: Non-discrimination
+
+No person shall be denied membership on the Leadership Team on the basis of race, sex, age, religion, national origin, sexual orientation, or any other basis prohibited by law.
+
+## Section 3: Powers of the leadership team
```

Comment:
There SHOULD be in the bylaws that we are a part of the GNOME Foundation and that the GNOME Foundation own the property (both physical and intellectual) of Black Python Devs and manage financial matters, and ultimately take responsibility in the organization's decisions as a 501.3c non-profit.

Reply:
This will best fit as a new section under Article I